components/src/core/HeroBanner/HeroBanner.tsx

```diff
@@ -16,20 +16,22 @@ export type HeroBannerClasses = {
     root?: string;
 };
 
-export type HeroBannerProps = {
+export type HeroBannerProps = React.HTMLAttributes<HTMLDivElement> & {
     classes?: HeroBannerClasses;
     divider?: boolean;
     limit?: number;
 };
 
-export const HeroBanner = (props: HeroBannerProps & any): JSX.Element => {
-    const { classes, divider, limit } = props;
+export const HeroBanner = (props: HeroBannerProps): JSX.Element => {
+    const { classes, divider, limit, ...otherDivProps } = props;
     const defaultClasses = useStyles(props);
     const isArray = Array.isArray(props.children);
     return (
         <React.Fragment>
```

Comment:
can we change `<React.Fragment>` to `<>` to be consistent with other components?